### PR TITLE
RHDEVDOCS-6198: Content creation for PSA labels note

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.15

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6198

Link to docs preview:

The second note under the section [Setting up an Argo CD Instance](https://85061--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance.html).
QE review:
- [x] QE has approved this change.

SME review: @svghadi 
QE review: @varshab1210 
Peer review @bergerhoffer 

Additional information:

### CODE REVIEW

**Review by svghadi — CHANGES_REQUESTED** [review]
Hi @Dhruv-Soni11 , the content is good. Can we just move it up in the same page?
image

**Comment by Dhruv-Soni11** [comment]
Hi @svghadi, I have incorporated your feedback in the content. PTAL. Thanks.

**Comment by svghadi** [comment]
/lgtm
Thanks @Dhruv-Soni11

**Comment by varshab1210** [comment]
LGTM, thanks

File: argocd_instance/setting-up-argocd-instance.adoc [hunk 1]
```diff
@@ -10,9 +10,11 @@ By default, {gitops-title} installs an instance of Argo CD in the `openshift-git
...
+For {gitops-shortname} version 1.13 and later:
 
-You can modify the route TLS termination policy by configuring the `.spec.server.route.tls` field of the Argo CD CR.
+* Route TLS termination is set as default to the `reencrypt` mode for both the default and user-defined Argo CD instances. TLS connections to the Argo CD instances now receive the default ingress certificate that is set in {OCP}, instead of the self-signed Argo CD certificate. You can modify the route TLS termination policy by configuring the `.spec.server.route.tls` field of the Argo CD CR.
+
+* Restrictive Pod Security Admission (PSA) labels are applied to the `openshift-gitops` namespace to ensure compliance with {OCP} standards. If you are running additional workloads, for example - monitoring or logging, in this namespace, ensure they comply with the restrictive PSA requirements. If compliance is not feasible, consider using a user-defined, cluster-scoped ArgoCD instance, where PSA labels are not applied or are controlled by the operator.
```
Comment (bergerhoffer) [inline argocd_instance/setting-up-argocd-instance.adoc:17]: - The profile is called "restricted", not "restrictive", so I assume you'd want to use that term? https://docs.openshift.com/container-platform/4.17/authentication/understanding-and-managing-pod-security-admission.html#psa-profiles_understanding-and-managing-pod-security-admission
- Pod security admission isn't capitalized
- When you say "the Operator", can you clarify which Operator you mean? 
- A few other suggestions

```suggestion
* Restricted pod security admission (PSA) labels are applied to the `openshift-gitops` namespace to ensure compliance with {OCP} standards. If you are running additional workloads in this namespace, such as monitoring or logging, ensure that they comply with the restricted PSA requirements. If compliance is not feasible, consider using a user-defined, cluster-scoped ArgoCD instance, where PSA labels are not applied or controlled by the Operator.
```

**Pull request merged**